### DISCUSSION TITLE
is this support Byobu(Multiplexer) Terminal.

### DISCUSSION THREAD

**Discussion by ios37** [migueldeicaza/SwiftTerm#299] [General]
is it possible to handle Multiplexer Terminal same like Byobu?

**Comment by ios37** [comment]
@migueldeicaza is it supported?

**Comment by ios37** [comment]
@migueldeicaza May you please check this. Thanks in advance.

**Comment by migueldeicaza** [comment]
No clue what this question is about, if you want to check if something works, you can check yourself.
